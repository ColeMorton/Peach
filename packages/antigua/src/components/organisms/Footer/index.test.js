import React from 'react'
import renderer from 'react-test-renderer'
import ComponentMock from './mocks'

it('renders correctly', () => {
  const tree = renderer.create(<ComponentMock />).toJSON()
  expect(tree).toMatchSnapshot()
})
