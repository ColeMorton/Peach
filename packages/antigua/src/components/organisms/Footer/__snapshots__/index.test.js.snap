// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui buttons sc-bwzfXH irATXA"
>
  <span
    className="sc-bxivhb kcsnNr sc-bdVaJa jEkClK"
  >
    <button
      className="ui fluid icon button"
      disabled={undefined}
      onClick={[Function]}
      role="button"
      tabIndex={undefined}
    >
      <a
        className="sc-htpNat hWqqCL"
        href="/home"
        onClick={[Function]}
      >
        <i
          aria-hidden="true"
          className="home big inverted icon"
        />
      </a>
    </button>
  </span>
  <span
    className="sc-bxivhb kcsnNr sc-bdVaJa jEkClK"
  >
    <button
      className="ui fluid icon button"
      disabled={undefined}
      onClick={[Function]}
      role="button"
      tabIndex={undefined}
    >
      <a
        className="sc-htpNat hWqqCL"
        href="/"
        onClick={[Function]}
      >
        <i
          aria-hidden="true"
          className="compass big inverted icon"
        />
      </a>
    </button>
  </span>
  <span
    className="sc-bxivhb kcsnNr sc-bdVaJa jEkClK"
  >
    <button
      className="ui fluid icon button"
      disabled={undefined}
      onClick={[Function]}
      role="button"
      tabIndex={undefined}
    >
      <a
        className="sc-htpNat hWqqCL"
        href="/scan"
        onClick={[Function]}
      >
        <i
          aria-hidden="true"
          className="camera retro big inverted icon"
        />
      </a>
    </button>
  </span>
  <span
    className="sc-bxivhb kcsnNr sc-bdVaJa jEkClK"
  >
    <button
      className="ui fluid icon button"
      disabled={undefined}
      onClick={[Function]}
      role="button"
      tabIndex={undefined}
    >
      <a
        className="sc-htpNat hWqqCL"
        href="/"
        onClick={[Function]}
      >
        <i
          aria-hidden="true"
          className="heart big inverted icon"
        />
      </a>
    </button>
  </span>
  <span
    className="sc-bxivhb kcsnNr sc-bdVaJa jEkClK"
  >
    <button
      className="ui fluid icon button"
      disabled={undefined}
      onClick={[Function]}
      role="button"
      tabIndex={undefined}
    >
      <a
        className="sc-htpNat hWqqCL"
        href="/"
        onClick={[Function]}
      >
        <i
          aria-hidden="true"
          className="user big inverted icon"
        />
      </a>
    </button>
  </span>
</div>
`;
